### PR TITLE
Teamsfx toolkit migration nodejs samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/tab-multi-hub/ts/teamsapp.local.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile:
+      titleId: M365_TITLE_ID
+      appId: M365_APP_ID
+
+deploy:
+  # Run npm command
+  - uses: cli/runNpmCommand
```
Comment (formulahendry) [inline samples/tab-multi-hub/ts/teamsapp.local.yml:93]: `npm install` for api-server as well?

Comment (Harikrishnan-MSFT) [reply samples/tab-multi-hub/ts/teamsapp.local.yml:93]: Fixed

**Review by formulahendry — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
@Harikrishnan-MSFT - Run build pipeline.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5688600032
image

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**